independent-publisher-connectors/xkcd/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},

```

Comment:
What are the connection parameters?

Reply:
There are no connection parameters required for this connection as it does not use authentication.